src/routes/wallet/wallet.routes.js

```diff
@@ -465,4 +466,45 @@ walletRouter.post(
  */
 walletRouter.post('/stop', stop);
 
+/**
+ * PUT request to mark inputs from a txHex as used.
+ * For the docs, see api-docs.js
+ */
+walletRouter.put(
+  '/utxos-selected-as-input',
+  checkSchema({
+    txHex: {
+      in: ['body'],
+      errorMessage: 'Invalid txHex',
+      isString: true,
+      custom: {
+        options: (value, { req, location, path }) => {
+          // Test if txHex is actually hex
+          if (!(/^[0-9a-fA-F]+$/.test(value))) return false;
```

Comment:
[question] We doesn't need to challenge the length?

Reply:
We don't know the length of the txHex.
We know that the length should be higher than 10 chars (5 bytes) and the ceiling is a few kilobytes (do not know the exact number) but it is so variable that there is no need to check the length.

Reply:
Got it!